submissions/sadoharu/task_js_dom_/scripts.js

```diff
@@ -0,0 +1,61 @@
+window.addEventListener("DOMContentLoaded", () => {
+  const pages = document.querySelectorAll(".section__page");
+  const navList = document.querySelector(".nav-bar__list");
+
+  const navListItem = [
+    {
+      title: "Overview",
+      icon: "icon-browser",
+    },
+    {
+      title: "About",
+      icon: "icon-about-dot-me",
+    },
+    {
+      title: "Achievement",
+      icon: "icon-award",
+    },
+    {
+      title: "Projects",
+      icon: "icon-files-empty",
+    },
+    {
+      title: "Reviews",
+      icon: "icon-rate_review",
+    },
+  ];
+
+  navListItem.forEach((item, i) => {
+    const liItem = document.createElement("li");
+    liItem.classList.add("nav-bar__list_item");
+    liItem.setAttribute("data-index", i);
+    liItem.innerHTML = `
+      <button class="nav-bar__list_button desktop">
+        <span class="${item.icon}">
+        </span>
+        ${item.title}
+      </button>
+      <button class="nav-bar__list_button mobile ${item.icon}"></button>
+    `;
+    navList.append(liItem);
+  });
+
+  navList.querySelector(".nav-bar__list_button").classList.add("active");
+
+  const togglePageVisibility = (show = 0) => {
```

Comment:
What Ivan meant is that this is unreadably and not intuitive. `show` doesn't tell anything about what kind of data is there and these numbers are basically random from this function perspective. You can call this variable something like `currentPage` to add more sense to it. There is also really no reason to make this function callable without passing an argument. 

About the `0` and other indexes. You're basically gathering an array of HTML elements that are already in your HTML file. But from what I see, you already figured out how to dynamically create elements from JS with your menu buttons. Why don't you do the same with pages?